Gemfile

```diff
@@ -55,6 +55,8 @@ gem 'surveyor', :git => 'https://github.com/caboteria/surveyor.git', branch: 'ra
 gem 'therubyracer', '0.10.2', :platforms => :ruby
 gem 'uglifier', '>= 1.0.3'
 gem 'will_paginate'
+gem 'omniauth-cas'
```

Comment:
Yes, let me look at the gem in the morning. It could be that it's just fine.

On Apr 18, 2016, at 6:46 PM, rkmadsen <notifications@github.com<mailto:notifications@github.com>> wrote:




In Gemfile<https://github.com/sparc-request/sparc-request/pull/349#discussion_r60147597>:

> @@ -55,6 +55,8 @@ gem 'surveyor', :git => 'https://github.com/caboteria/surveyor.git', branch: 'ra
>  gem 'therubyracer', '0.10.2', :platforms => :ruby
>  gem 'uglifier', '>= 1.0.3'
>  gem 'will_paginate'
> +gem 'omniauth-cas'


We'll can look at possibly taking over that gem or writing a new implementation but for now would it be possible to include it as an option for CAS auth?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub<https://github.com/sparc-request/sparc-request/pull/349/files/7b28d9eaf48809083d026887bf07a4fac3575360#r60147597>